tutoriale/Instalacja_Anakondy_W10-11.md

```diff
@@ -0,0 +1,56 @@
+# Anaconda
```

Comment:
czemu w nazwie pliku W10_11?

Reply:
ponieważ tutorial miał być przeznaczony dla Windowsa

Reply:
Lepiej napisać to na początku w dokumencie niż dawać magiczne tagi w nazwie